### PR TITLE
chore: Unify migration histories

### PR DESCRIPTION
**Pull request by ununhexium**
https://github.com/sovity/PMO-Software/issues/1103

```[tasklist]
### Checklist
- [x] The PR title is short and expressive.
- [x] I have updated the CHANGELOG.md. See [changelog_update.md](https://github.com/sovity/authority-portal/tree/main/docs/dev/changelog_updates.md) for more information.
- [x] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [ ] I have updated the Community Edition [Postman-Collection](https://github.com/sovity/edc-extensions/blob/main/docs/api/postman_collection.json) if I changed existing APIs or added new APIs (e.g. for Management-API or API-Wrapper)
- [x] I have performed a **self-review**
```

### CODE REVIEW

File: extensions/postgres-flyway/src/main/resources/db/migration/V2__Delete-Transfer-Processes-Trigger.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
+-- Required for reasonably fast ON DELETE CASCADE from edc_transfer_process
```
Comment (kamilczaja) [inline extensions/postgres-flyway/src/main/resources/db/migration/V2__Delete-Transfer-Processes-Trigger.sql:1]: Copyright header missing

**Review by kamilczaja — CHANGES_REQUESTED** [review]

File: extensions/postgres-flyway/src/main/resources/db/migration/V2__Delete-Transfer-Processes-Trigger.sql [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    where i.transferprocess_id = o.transferprocess_id and c.count > 3000;
+
+    return null;
+end;
+$$ language plpgsql;
+
+create trigger delete_old_rows after insert
+    on edc_transfer_process
+    for each row
+execute function transfer_process_delete_old_rows();
```
[the remark this answers is not in the document]
Comment (ununhexium) [reply extensions/postgres-flyway/src/main/resources/db/migration/V2__Delete-Transfer-Processes-Trigger.sql:41]: I would like to keep the same content, to keep the same checksum as what we currently have in the exiting flyway migration, to not confuse people if they check their migrations.

https://github.com/sovity/edc-extensions/blob/main/extensions/postgres-flyway/src/main/resources/migration/default/V2__Delete-Transfer-Processes-Trigger.sql

File: extensions/postgres-flyway/src/main/resources/db/migration/V2__Delete-Transfer-Processes-Trigger.sql (hunk 1 above)
Comment (ununhexium) [reply extensions/postgres-flyway/src/main/resources/db/migration/V2__Delete-Transfer-Processes-Trigger.sql:1]: Same argument as above.

**Review by kamilczaja — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,8 +17,13 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 
 #### Patch Changes
 
+- Unify database migrations' histories
```
Comment (kamilczaja) [inline CHANGELOG.md:20]: Past tense to stay consistent with the rest of the Changelog
```suggestion
- Unified database migration histories
```

**Review by kamilczaja — CHANGES_REQUESTED** [review]

**Review by kamilczaja — APPROVED** [review]
LGTM

**Pull request merged**